.circleci/config.yml

```diff
@@ -0,0 +1,286 @@
+version: 2.1
+
+workflows:
+  celery:
+    jobs:
+      - build-and-upload
+      - run-unit-tests:
+          requires:
+            - build-and-upload
+      - code-coverage-report:
+          requires:
+            - run-unit-tests
+
+orbs:
+  gcp-gke: circleci/gcp-gke@0.2.0
+  slack: circleci/slack@3.2.0
+
+commands:
+  configure_environment:
+    description: "Initial environment setup: Configure Gcloud and conditionally installs Docker Compose."
+    parameters:
+      with_compose:
+        description: "If true, sets up Docker Compose, which can then be used to
+        spin up Docker containers."
+        type: boolean
+        default: false
+    steps:
+      - run:
+          name: Authorize gcloud
+          command: |
+            echo $GCLOUD_SERVICE_KEY > ${HOME}/gcloud-service-key.json
+            gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
+            gcloud --quiet config set project sightmachine-178216
+            gcloud --quiet config set compute/zone us-west1-a
+            gcloud --quiet container clusters get-credentials dev-k8s-uw1
+            ## To authenticate to Container Registry, use gcloud as a Docker credential helper
+            echo y | gcloud auth configure-docker
+
+      - when:
+          condition: << parameters.with_compose >>
+          steps:
+            - run:
+                name: Setup Docker Compose
+                command: |
+                  curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+                  chmod +x ~/docker-compose
+                  mv ~/docker-compose /usr/local/bin/docker-compose
+
+  send_slack_msg:
+    description: "If the job fails for the master branch, send a message to the Slack channel."
+    steps:
+      - slack/status:
+          failure_message: Uh-oh! *$CIRCLE_PROJECT_REPONAME* (master) - Job Failed <$CIRCLE_BUILD_URL|#$CIRCLE_BUILD_NUM>
+          include_project_field: false
+          include_job_number_field: false
+          fail_only: true
+          only_for_branches: master
+          webhook: $SLACK_WEBHOOK_URL
+
+jobs:
+  ## ------------------ Build and Upload Celery Docker image to GCP Container Registry ------------------
+
+  build-and-upload:
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
```

Comment:
We can probably disable this to save money, since I don't think we're in a rush for Celery CI builds to complete.